java/core/src/test/org/apache/orc/TestColumnStatistics.java

```diff
@@ -699,6 +703,43 @@ public void testDecimalMinMaxStatistics() throws Exception {
         "Incorrect minimum value");
   }
 
+  @Test
+  public void testDoubleColumnWithoutDoubleStatistics() throws Exception {
```

Comment:
Shall we move `testDoubleColumnWithoutDoubleStatistics` to `TestRecordReaderImpl`?

Reply:
Let's refactor this pr later